### PR TITLE
Allow certain mobs to be var-edited to become completely docile.

### PR DESCRIPTION
**Pull request by Mercenaryblue**
[Changelogs]

:cl:
balance: Certain mobs can now be var-edited to become completely docile.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Certain mobs can be var-edited to become completely docile. (ie: simple_animal/hostile/retaliate)
*They no longer look around for enemies.
*They no longer retaliate against friendly fire.
*They still take their anger on furniture, so beware.

Why?  Because players likes to find and keep weird pets, and their behavior can be very erratic,.
Even with the faction system, there will be trouble. For example, I saw a pet watcher kill a vampire player in bat form in one shot. I doubt he was pleased about it. It gives admins and coders more leeway to what creatures they want to feature in their events and/or items.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/retaliate/retaliate.dm [hunk 1]
```diff
@@ -1,7 +1,10 @@
 /mob/living/simple_animal/hostile/retaliate
 	var/list/enemies = list()
+	var/docile = FALSE
```
Comment (Qustinnus) [inline code/modules/mob/living/simple_animal/hostile/retaliate/retaliate.dm]: why not on hostile level

**Comment by KorPhaeron** [comment]
@vuonojenmustaturska Is this the best way to do this?I thought you already had a way to turn off AI behavior

**Comment by ShizCalev** [comment]
Wouldn't you just have to set their faction to neutral?

**Comment by Mercenaryblue** [comment]
They'll still attack anything not in their faction then.
This puts in some extra checks.

**Comment by ExcessiveUseOfCobblestone** [comment]
doesn't AI status cover this?

**Comment by Mercenaryblue** [comment]
Right. I'll just close it then.

**Comment by vuonojenmustaturska** [comment]
AIStatus doesn't completely cover this, AI_OFF just means neither npcpool nor idlenpcpool subsystems will process the mob. The mob will sit in place doing absolutely nothing. It's also not vareditable, it's more of a read-only var if we had those. You're supposed to use toggle_ai(AI_ON|AI_IDLE|AI_OFF)

**Pull request closed without merging**